### DISCUSSION TITLE
Disable Acceleration Control

### DISCUSSION THREAD

**Discussion by mmunchinski** [OrcaSlicer/OrcaSlicer#789] [Q&A]
I am currently using a FLSun V400 delta printer and am using the 1.6.2-beta release.  I typically disable acceleration control in the slicer and let Klipper handle it.  I am finding that when I set Acceleration to 0 in orca, all M204 commands disappear except for one single M204 command at the end of the first layer that seems to correlate to travel speed.  Is this a bug or intended feature?  Is there any way to turn off any aspect of acceleration control within Orca?

**Comment by SoftFever** [comment]
Hi @mmunchinski 
Can you raise a tick(bug) for this?

**Reply by mmunchinski** [reply]
Done #795

**Comment by vgdh** [comment]
@mmunchinski 
Should this thread be closed since https://github.com/SoftFever/OrcaSlicer/issues/795 is resolved?
